packages/react-chat/src/contexts/RuntimeContext/useRuntimeState.ts

```diff
@@ -21,14 +23,15 @@ import { createContext, useRuntimeAPI } from './useRuntimeAPI';
 export interface Settings {
   assistant: AssistantOptions;
   config: ChatConfig;
+  traceHandlers?: TraceDeclaration<RuntimeMessage, any>[];
 }
 
 const DEFAULT_SESSION_PARAMS = {
   turns: [],
   startTime: Date.now(),
 };
 
-export const useRuntimeState = ({ assistant, config }: Settings) => {
+export const useRuntimeState = ({ assistant, config, traceHandlers }: Settings) => {
```

Comment:
allowing `traceHandlers` to be passed here so that new traces handlers can be added easily when using `react-chat` as a library